src/base/static/components/molecules/map-legend-item.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import PropTypes from "prop-types";
+import classNames from "classnames";
+
+import { Icon, Swatch } from "../atoms/feedback";
+import { PanelLabel } from "../atoms/typography";
+
+import "./map-legend-item.scss";
+
+const MapLegendItem = props => (
+  <div className="map-legend-item">
+    {props.icon && (
+      <div className="map-legend-item__icon-container">
```

Comment:
I thought we were going to use CSS-in-JS for all new code moving forward? Is there a reason we aren't doing that here?

Reply:
I'll refactor this for CSS-in-JS.